extensions/wikia/TemplateClassification/TemplateClassificationView.php

```diff
@@ -20,17 +22,27 @@ public function renderEditableType( $fallbackMsg, $user ) {
 			}
 			return $fallbackMsg;
 		}
-
 	}
 
 	/**
 	 * Mock for frontend work
 	 */
-	private function getTemplateType() {
+	private function getTemplateType( $articleId ) {
+		$response = \F::app()->sendRequest( 'TemplateClassificationMockApiController',
+			'getTemplateType',
+			[
+				'articleId' => $articleId,
+			],
+			true, \WikiaRequest::EXCEPTION_MODE_RETURN
+		);
+		$templateTypeKey = $response->getData()[ \TemplateClassificationMockApiController::TYPE_FIELD ];
 		/* Messages generated by code below
 		 * template-classification-type-unclassified
+		 * template-classification-type-infobox
+		 * template-classification-type-navbox
+		 * template-classification-type-quote
 		 */
-		return wfMessage( 'template-classification-type-'.\TemplateClassification::TEMPLATE_UNCLASSIFIED )->escaped();
+		return wfMessage( 'template-classification-type-' . $templateTypeKey )->escaped();
```

Comment:
You can go with double quotes and `"template-classification-type-{$templateTypeKey}"` but it's minor.